anchors/bootstrapper.go

```diff
@@ -29,6 +30,9 @@ func (Bootstrapper) Bootstrap(ctx map[string]interface{}) error {
 	}
 	client := ctx[ethereum.BootstrappedEthereumClient].(ethereum.Client)
 
+	fmt.Println("anchors")
```

Comment:
I know this is testing material, let's try to keep as "production" as possible, bc otherwise when we merge to develop it will be harder to catch.

Reply:
fixed in next PR